Cannot install the python binding

## Steps to reproduce

```bash
conda create -n test python==3.11 # conda is only used to control python version
conda activate test
pip install libmata
```

## Expected behavior

Successfully installed the library.

## Actual behavior

Error when compiling the binding. It should be noted that `<error>` probably means the generated file is wrong.

```txt
      libmata/nfa/nfa.cpp: In function ‘PyObject* __pyx_gb_7libmata_3nfa_3nfa_3Nfa_52generator(__pyx_CoroutineObject*, PyThreadState*, PyObject*)’:
      libmata/nfa/nfa.cpp:12847:42: error: expected primary-expression before ‘<’ token
      12847 |   __pyx_cur_scope->__pyx_v_transitions = <error>;
            |                                          ^
      libmata/nfa/nfa.cpp:12847:43: error: ‘error’ was not declared in this scope; did you mean ‘perror’?
      12847 |   __pyx_cur_scope->__pyx_v_transitions = <error>;
            |                                           ^~~~~
            |                                           perror
      libmata/nfa/nfa.cpp:12847:49: error: expected primary-expression before ‘;’ token
      12847 |   __pyx_cur_scope->__pyx_v_transitions = <error>;
            |                                                 ^
      libmata/nfa/nfa.cpp: In function ‘PyObject* __pyx_pf_7libmata_3nfa_3nfa_3Nfa_57get_trans_as_sequence(__pyx_obj_7libmata_3nfa_3nfa_Nfa*)’:
      libmata/nfa/nfa.cpp:13551:27: error: expected primary-expression before ‘<’ token
      13551 |   __pyx_v_c_transitions = <error>;
            |                           ^
      libmata/nfa/nfa.cpp:13551:28: error: ‘error’ was not declared in this scope; did you mean ‘perror’?
      13551 |   __pyx_v_c_transitions = <error>;
            |                            ^~~~~
            |                            perror
      libmata/nfa/nfa.cpp:13551:34: error: expected primary-expression before ‘;’ token
      13551 |   __pyx_v_c_transitions = <error>;
            |                                  ^
      error: command '/usr/bin/g++' failed with exit code 1

```

--- a/bindings/python/libmata/nfa/nfa.pxd
+++ b/bindings/python/libmata/nfa/nfa.pxd
@@ -64,7 +64,6 @@ cdef extern from "mata/nfa/nfa.hh" namespace "mata::nfa":
 
     cdef cppclass CDelta "mata::nfa::Delta":
         vector[CStatePost] state_posts
-        CTransitions transitions
 
         void reserve(size_t)
         CStatePost& state_post(State)

